src/SnmpRequest.h

```diff
@@ -10,8 +10,11 @@
 #define SQUID_SNMPREQUEST_H_
 
 #if SQUID_SNMP
+#include "ip/Address.h"
 #include "snmp_session.h"
 
+class ACLChecklist;
```

Comment:
Rule of thumb: Include forward.h that forward-declares X (instead of duplicating that forward-declaration).

```suggestion
#include "acl/forward.h"
#include "ip/Address.h"
#include "snmp_session.h"
```